istioctl/cmd/istioctl/handlers.go

```diff
@@ -0,0 +1,28 @@
+// Copyright 2018 Istio Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package main
+
+import (
+	"strings"
+)
+
+func getProxyDetails(proxyName, namespace string) (string, string) {
```

Comment:
```suggestion
func inferPodInfo(proxyName, namespace string) (string, string) {
```

prefer this name, more accurate ^^

Reply:
Thanks @liamawhite taken your suggestions and made modifications. Thanks for taking time for review.